packages/extension-chakra-storefront/src/components/product-scroller/index.test.js

```diff
@@ -48,15 +47,20 @@ describe('Product Scroller', () => {
         expect(screen.getByTestId('custom-header')).toBeInTheDocument()
     })
     test('Renders left/right scroll buttons', async () => {
-        const user = userEvent.setup()
-        renderWithProviders(<ProductScroller title="Scroller Title" products={testProducts} />)
-        await user.click(screen.getByTestId('product-scroller-nav-right'))
+        const {user} = renderWithProviders(
+            <ProductScroller title="Scroller Title" products={testProducts} />
+        )
+        await act(async () => {
+            await user.click(screen.getByTestId('product-scroller-nav-right'))
+        })
         expect(window.HTMLElement.prototype.scrollBy).toHaveBeenCalledWith({
             top: 0,
             left: 0,
             behavior: 'smooth'
         })
-        await user.click(screen.getByTestId('product-scroller-nav-left'))
+        await act(async () => {
```

Comment:
Wrap user events inside act to resolve the React warning issues. Doing this reduce 90% of the warnings. Some may need more attention since it may not related to the component being tested.